README.md

```diff
@@ -31,5 +31,38 @@ Include diagrams in the site:
 * Export diagrams via http://localhost:8090
 * Save exported diagrams to the relevant folder in `src/markdown-pages`
 * Update the relevant markdown file to incorporate diagrams into the site.
-* 
 * `git commit` changes
+
+### How to Run the application locally
+
+To run Ha Kākano locally, you will need to start the shared infrastructure services and then start the individual applications. 
+
+#### Shared Infrastructure
+
+You will need at least the following installed on your machine before you start:-
+* A [JVM](https://aws.amazon.com/corretto/?filtered-posts.sort-by=item.additionalFields.createdDate&filtered-posts.sort-order=desc) installed on the target machine in order to run the [Batect](https://batect.dev/) tool.
+* [Docker](https://docs.docker.com/get-docker/)
+* And of course a **git** tool of some sort to clone the repo to your machine
+
+Shared services will expose the following ports that will need to be available before you start:-
+* 5432 for Postgres
+* 8080 for the Manager API
+* 9092 for Kafka
+
+With above in place, clone the repo to your machine and do the following from a command line shell to start the shared services:-
+1. `cd packages/Manager`
+2. `./batect runSupportServices`
+
+_And from a new shell session in the same folder_
+
+3. `./gradlew bootRun`
+
+#### Application Packages
+
+Having started the shared infrastructure, you will find specific run instructions for each application package in its own README.md file.  For example, [here are the run instructions for the Plan Limits UI](./packages/PlanLimitsUI/README.md).
+
+## `start.sh` convenience script
+
+To simplify running EOP locally, a `start.sh` script has been provided in the root of the repository.  This script will start the shared infrastructure services and then another application component that you name as an argument.
+
+For example, to start the PlanUnitsUI application, you would run `./start.sh PlanUnitsUI`.  
```

Comment:
If I understand the start script correctly, you can provide multiple arguments for each thing you want to start? (nice) but maybe that needs to be talked about here?

Reply:
No, it doesn't yet allow for starting multiple things, just one.  It is mostly a quick way of starting the PlanLimitsUI to begin with and/or restart the backend services.  I hope to improve it over time.  I'll tweak the readme to make its usage clearer.

Reply:
Ah sweet, yeah it was actually the "$2 $3 $4 $5 ..." I saw in the script that made me think it was doing stuff with multiple args, if not, then its fine

Reply:
TBH has a few problems I think relating to docker loading and unloading times, so def a work in progress.  Works well after first machine startup